model.go

```diff
@@ -10,3 +10,18 @@ type Paging struct {
 	PerPage int64 `json:"perPage"`
 	Offset  int64 `json:"offset"`
 }
+
+type UpdateSearchConcurrencySettingsScheduleReq struct {
+	MaxSearchesPer *int
+	AutoSummaryPer *int
+}
+
+func (u *UpdateSearchConcurrencySettingsScheduleReq) SetMaxSearchesPer(value int) *UpdateSearchConcurrencySettingsScheduleReq {
+	u.MaxSearchesPer = &value
+	return u
+}
+
+func (u *UpdateSearchConcurrencySettingsScheduleReq) SetAutoSummaryPer(value int) *UpdateSearchConcurrencySettingsScheduleReq {
+	u.AutoSummaryPer = &value
+	return u
+}
```

Comment:
Considering the structure is simple, I don't think we need setters.
```suggestion
```

Reply:
The setter are helpful if the properties are pointer if you prefer value I ll remove let me know

Reply:
removed